qutebrowser/keyinput/modeparsers.py

```diff
@@ -230,3 +230,59 @@ def __init__(self, win_id, parent=None):
         super().__init__(win_id, parent, supports_count=True,
                          supports_chains=True)
         self.read_config('caret')
+
+
+class MarkKeyParser(keyparser.BaseKeyParser):
+
+    """KeyParser for set_mark and jump_mark mode.
+
+    Attributes:
+        _mode: Either KeyMode.set_mark or KeyMode.jump_mark.
+    """
+
+    def __init__(self, win_id, mode, parent=None):
+        super().__init__(win_id, parent, supports_count=False,
+                         supports_chains=False)
+        self._mode = mode
+
+    def handle(self, e):
+        """Override handle to always match the next key and create a mark.
+
+        Args:
+            e: the KeyPressEvent from Qt.
+
+        Return:
+            True if event has been handled, False otherwise.
+        """
+        if utils.keyevent_to_string(e) is None:
+            # this is a modifier key, let it pass and keep going
+            return True
+
+        key = e.text()
+
+        if not key.isalpha() and key != "'":
```

Comment:
I guess not, unless you wanted to reserve some other marks for special meanings.

In vim, marks 0-9 are reserved for the almost hilarious feature of "your
position at the nth last time of exiting vim." So `'6` will jump me to my cursor
position at the 6th last time I exited vim. Useful!

There are some legitimately useful ones, like `]` and `[`, which cycle through
your marks in order. So you might mark several interesting places in a long
article and cycle through them.

Though now that I think about it, you can still manually overwrite these. So yeah, lets let anything be assigned as a mark